deny.toml

```diff
@@ -0,0 +1,55 @@
+[sources.allow-org]
+github = [
+    "mintlayer", #allow any code from mintlayer's github
+]
+
+[licenses]
+#we reject code without a license
+unlicensed = "deny"
+confidence-threshold = 0.92
+allow = [
+    "Apache-2.0",
+    "BSD-2-Clause",
+    "BSD-3-Clause",
+    "BSL-1.0",
+    "CC0-1.0",
+    "ISC",
+    "LicenseRef-ring",
+    "LicenseRef-webpki",
+    "MIT",
+    "MPL-2.0",
+    "Unicode-DFS-2016",
+    "Unlicense", #this is a specific license rather than no license at all
+    "Zlib",
+] #deny a license not in this set of licenses
+
+[[licenses.clarify]]
+name = "ring"
+expression = "LicenseRef-ring"
+license-files = [
+    { path = "LICENSE", hash = 0xbd0eed23 },
+]
+
+[[licenses.clarify]]
+name = "webpki"
+expression = "LicenseRef-webpki"
+license-files = [
+    { path = "LICENSE", hash = 0x001c7e6c },
+]
+
+[advisories]
```

Comment:
This won't run as is. You need a `check advisories` call too

Reply:
I pushed a change to also raise a job for "check advisories". However, the next PR should be merged earlier to avoid the issue mentioned by @TheQuantumPhysicist

https://github.com/mintlayer/mintlayer-core/pull/676